src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/TypeTypeVisitor.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Visitors
+{
+    /// <summary>
+    /// This represents the type visitor for <see cref="Type"/>.
+    /// </summary>
+    public class TypeTypeVisitor : TypeVisitor
+    {
+        /// <inheritdoc />
+        public TypeTypeVisitor(VisitorCollection visitorCollection) : base(visitorCollection)
+        {
+        }
+
+        /// <inheritdoc />
+        public override bool IsVisitable(Type type)
+        {
+            return type == typeof(Type);
+        }
+
+        /// <inheritdoc />
+        public override bool IsParameterVisitable(Type type)
+        {
+            return false;
+        }
+
+        /// <inheritdoc />
+        public override bool IsNavigatable(Type type)
+        {
+            return false;
+        }
+
+        /// <inheritdoc />
+        public override bool IsPayloadVisitable(Type type)
+        {
+            return false;
+        }
+
+        /// <inheritdoc />
+        public override void Visit(IAcceptor acceptor, KeyValuePair<string, Type> type, NamingStrategy namingStrategy, params Attribute[] attributes)
+        {
+            this.Visit(acceptor, name: type.Key, title: null, dataType: "string", dataFormat: null, attributes: attributes);
```

Comment:
I think its `dataType` value should be `"object"` instead of `"string"`.

Reply:
> I think its `dataType` value should be `"object"` instead of `"string"`.

Hi @justinyoo , The reason I use string instead of object when the response model serialized to json,
this property turning into string field but if you want i can change it

by the way thanks for the usefull package

Reply:
Oh, does it become string? Then, it makes sense to me. Would you be able to take a screenshot for that for confirm?

Reply:
Could you also check whether it becomes the "object", if you set the dataType value to "object"?

Reply:
![Screenshot_6](https://user-images.githubusercontent.com/17519791/122205313-c7fce780-cea8-11eb-991e-2a924925ba58.png)

After serialization
![Screenshot_7](https://user-images.githubusercontent.com/17519791/122205340-cf23f580-cea8-11eb-9913-5035ba0ea236.png)


@justinyoo

Reply:
Oh, I see. Is the value, "ConsoleApp3.Program, ConsoleApp3,  Version..." what you were expecting?

Reply:
> Oh, I see. Is the value, "ConsoleApp3.Program, ConsoleApp3, Version..." what you were expecting?

yes i expected this value

Reply:
Yep, let's ship it, then.